docs/source/55_api.md

```diff
@@ -339,6 +342,130 @@ val bcastFeatures = sc.loadFeatures("my/features.adam").broadcast()
+be expanded from.
+
+###### Filter Genotypes by Features
+
+This query joins an RDD of Genotypes against an RDD of Features using an inner
+join. The inner join will result in an RDD of key-value pairs, where the key is
+a genotype and the value is the feature that it overlaps. Because this is an
+inner join, records from either dataset that don't pair to the other are
+automatically dropped, providing the filter we are interested in. This query is
+useful for trying to identify genotypes that overlap features of interest. For
+example, if our feature file contains all the exonic regions of the genome,
+this query would extract all genotypes that fall in exonic regions.
+
+```scala
+// Inner join will filter out genotypic data not represented in the feature dataset
+val genotypes = sc.loadGenotypes("my/genotypes.adam")
+val features = sc.loadFeatures("my/features.adam")
+
+// We can use ShuffleRegionJoin…
+val joinedGenotypesShuffle = genotypes.shuffleRegionJoin(features)
+
+// …or BroadcastRegionJoin
+val joinedGenotypesBcast = features.broadcastRegionJoin(genotypes)
+
+// In the case that we only want Genotypes, we can use a simple projection
+val filteredGenotypesShuffle = joinedGenotypesShuffle.rdd.map(_._1)
+
+val filteredGenotypesBcast = joinedGenotypesBcast.rdd.map(_._2)
+```
+
+After the join, we can perform a transform function on the resulting RDD to
+manipulate it into providing the answer to our question. Since we were
+interested in the `Genotype`s that overlap a `Feature`, we map over the tuples
+and select just the `Genotype`.
+
+The difference between the ShuffleRegionJoin and BroadcastRegionJoin strategies
+is that a broadcast join sends the left dataset to all executors. In this case,
+we chose to send the `features` dataset because feature data is usually smaller
+in size than genotypic data.
+
+###### Group overlapping variant data by the gene they overlap
+
+This query joins an RDD of Variants against an RDD of Features, and immediately
+performs a group-by on the Feature. This produces an RDD whose elements are a
+tuple containing a Feature, and all of the Variants overlapping the Feature.
+This query is useful for trying to identify annotated variants that may
+interact (identifying frameshift mutations within a transcript that may act as
+a pair to shift and then restore the reading frame) or as the start of a query
+that computes variant density over a set of genomic features.
+
+```scala
+// Inner join with a group by on the features
+val features = sc.loadFeatures("my/features.adam")
+val variants = sc.loadVariants("my/variants.adam")
+
+// As a ShuffleRegionJoin, it can be implemented as follows:
+val variantsByFeatureShuffle = features.shuffleRegionJoinAndGroupByLeft(variants)
+
+// As a BroadcastRegionJoin, it can be implemented as follows:
+val variantsByFeatureBcast = variants.broadcastRegionJoinAndGroupByRight(features)
+```
+
+When we switch join strategies, we need to change the dataset that is on the
```

Comment:
Nit: I think this text would read a bit better if "to change the dataset that is" was replaced with "to swap which dataset is"